src/measure_theory/lp_space.lean

```diff
@@ -512,22 +512,64 @@ begin
   exact snorm'_add_le hf hg hq1_real,
 end
 
-section second_countable_topology
-variable [topological_space.second_countable_topology E]
+lemma snorm_add_lt_top_of_one_le {f g : α → E} (hf : mem_ℒp f q μ) (hg : mem_ℒp g q μ)
```

Comment:
When you have a lemma which is strictly superseded by a later lemma, you can mark it as private to indicate to the reader that it shouldn't be used:
```suggestion
private lemma snorm_add_lt_top_of_one_le {f g : α → E} (hf : mem_ℒp f q μ) (hg : mem_ℒp g q μ)
```